### PR TITLE
[Xcode13] Update all already publish frameworks to Xcode13 GA.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

**Review by chamons — APPROVED** [review]

File: src/avfoundation.cs [hunk 1]
```diff
@@ -9269,15 +9269,16 @@ interface AVAssetExportPresetApple {
 
 		[Field ("AVAssetExportPresetAppleM4V720pHD")]
 		NSString M4V720pHD { get; }
-
+#else 
+		[NoiOS]
 		[Field ("AVAssetExportPresetAppleM4V1080pHD")]
 		NSString M4V1080pHD { get; }
```
Comment (rolfbjarne) [inline src/avfoundation.cs:9275]: This means you're removing `M4V1080pHD` and `ProRes422Lpcm` from macOS, which is a breaking change.

Any particular reason why you can't remove the `#if MONOMAC` completely, and add `[NoXYZ]` to each field?

File: src/CoreAnimation/CAStructs.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+#if !COREBUILD
+    [Field ("CAFrameRateRangeDefault", "CoreAnimation")]
+    public static CAFrameRateRange Default
+        => (CAFrameRateRange) Marshal.PtrToStructure (Dlfcn.GetIndirect (Libraries.CoreAnimation.Handle, "CAFrameRateRangeDefault"), typeof (CAFrameRateRange))!;
+#endif
+
+		[DllImport (Constants.CoreAnimationLibrary)]
+		[return: MarshalAs (UnmanagedType.I1)]
+		static extern bool CAFrameRateRangeIsEqualToRange (CAFrameRateRange range, CAFrameRateRange other);
```
Comment (rolfbjarne) [inline src/CoreAnimation/CAStructs.cs:41]: Missing manual tests for manual API

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: src/avfoundation.cs (hunk 1 above)
Comment (mandel-macaque) [reply src/avfoundation.cs:9275]: I'm stupid, that should have been and #endif but is better with the NoXYZ attrs

File: src/AVFoundation/Enums.cs [hunk 3]
```diff
@@ -1358,4 +1358,25 @@ public enum AVCaptionConversionValidatorStatus : long
 		Stopped = 3,
 	}
 
+	[NoWatch, NoTV, MacCatalyst (15,0), Mac (12,0), iOS (15,0)]
+	[Flags]
+	[Native]
+	public enum AVCapturePrimaryConstituentDeviceRestrictedSwitchingBehaviorConditions : ulong
```
Comment (tj-devel709) [inline src/AVFoundation/Enums.cs:1364]: That name though lol

File: src/CoreAnimation/CAStructs.cs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+#endif
+	[StructLayout (LayoutKind.Sequential)]
+	public struct CAFrameRateRange
+	{
+
+		public float Minimum;
+		public float Maximum;
+		public float Preferred;
+
+    public CAFrameRateRange (float min, float max, float preferred) {
```
Comment (tj-devel709) [inline src/CoreAnimation/CAStructs.cs:27]: Maybe some spacing differences here or in the fields above it?

**Review by rolfbjarne — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Comment by dalexsoto** [comment]
Merging since test got fixed and was the only failure point.

**Pull request merged**